app/assets/stylesheets/base/defaults.scss

```diff
@@ -108,7 +108,7 @@ a {
     color: $white;
     display: inline-block;
     text-transform: uppercase;
-    font-family: 'proxima-nova', sans-serif;
+    font-family: "proxima-nova", verdana, sans-serif;
```

Comment:
@waidelacruz Did you push your changes? I don't see the new font-family variable.